crates/holochain/tests/inline_zome_sketch.rs

```diff
@@ -0,0 +1,178 @@
+        Default::default(),
+        Default::default(),
+        Default::default(),
+    );
+
+    InlineZome::new("", vec![entry_def.clone()])
+        .callback("create", move |api, ()| {
+            let entry_def_id: EntryDefId = entry_def.id.clone();
+            let entry = Entry::app(().try_into().unwrap()).unwrap();
+            let hash = api.create((entry_def_id, entry))?;
+            Ok(hash)
+        })
+        .callback("read", |api, hash: HeaderHash| {
+            api.get((hash.into(), GetOptions::default()))
+                .map_err(Into::into)
+        })
+}
+
+#[tokio::test(threaded_scheduler)]
+#[cfg(feature = "test_utils")]
+async fn extremely_verbose_inline_zome_sketch() -> anyhow::Result<()> {
+    let envs = test_environments();
+
+    // Bundle the single zome into a DnaFile
+
+    let (dna_file, zome) = DnaFile::unique_from_inline_zome("zome1", simple_crud_zome()).await?;
+    let dna_hash = dna_file.dna_hash().clone();
+
+    // Get two agents
+
+    let (alice, bobbo) = TestAgent::two(envs.keystore()).await;
+    let alice_cell_id = CellId::new(dna_hash.clone(), alice.clone());
+    let bobbo_cell_id = CellId::new(dna_hash.clone(), bobbo.clone());
+
+    // Create a Conductor
+
+    let conductor = Conductor::builder().test(&envs).await?;
+
+    // Install the DNA
+
+    conductor.install_dna(dna_file).await?;
+
+    // Install and activate one app for Alice and another for Bob
+    // TODO: develop tools to app installation much less verbose
+
+    conductor
+        .clone()
+        .install_app(
+            "app:alice".to_string(),
+            vec![(
+                InstalledCell::new(alice_cell_id.clone(), "dna".into()),
+                None,
+            )],
+        )
+        .await?;
+    conductor
+        .clone()
+        .install_app(
+            "app:bobbo".to_string(),
+            vec![(
+                InstalledCell::new(bobbo_cell_id.clone(), "dna".into()),
+                None,
+            )],
+        )
+        .await?;
+    conductor.activate_app("app:alice".to_string()).await?;
+    conductor.activate_app("app:bobbo".to_string()).await?;
+    conductor.clone().setup_cells().await?;
+
+    // Call the "create" zome fn on Alice's app
+    // TODO: develop tools to make zome calls much less verbose
+
+    let hash: HeaderHash = {
+        let response = conductor
+            .call_zome(ZomeCallInvocation {
+                cell_id: alice_cell_id.clone(),
+                zome: zome.clone(),
+                fn_name: "create".into(),
+                payload: ExternInput::new(().try_into().unwrap()),
+                cap: None,
+                provenance: alice.clone(),
+            })
+            .await??;
+        unwrap_to!(response => ZomeCallResponse::Ok)
+            .clone()
+            .into_inner()
+            .try_into()?
+    };
+
+    // Wait long enough for Bob to receive gossip
+
+    tokio::time::delay_for(std::time::Duration::from_millis(500)).await;
```

Comment:
I don't want to calculate the number of DhtOps I need to wait for, can you tell me? ;)

Reply:
if you run `wait_for_integration()` with 1000 expected values it and do `RUST_LOG=[wait_for_integration]=debug` it will tell you the count it's seeing :) then plug that number into `wait_for_integration()`

Reply:
@freesig for some reason I can't get this integration to emit any output, so this is a rabbit hole it's not worth going down now. Also, I feel that we need a more robust way to `wait_for_integration` that's not based on trial and error. I am having an idea that goes: once we hook up some system signals like DhtOpIntegrated, we can just count those and compare them between nodes. Or even just compare each node's integrated count with the sum of all other agents' authored counts -- if the former if ever lower than the latter, then we're not consistent yet.

Reply:
So, I want to merge this now and update that test later. It hasn't flaked out yet. In fact, it works more than half the time with no delay at all, so I think 500ms is pretty solid for this test.